omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -532,7 +532,7 @@
 # PostgreSQL
 ###
 # For now, we're hardcoding the version directory suffix here:
-default['private_chef']['postgresql']['version'] = '9.6'
+default['private_chef']['postgresql']['version'] = '13.3'
```

Comment:
There seem to be muiltiple places in the file that are still pointing to 9.6 (in comments etc)

Reply:
I just looked.  The comments with 9.6 refer to specific postgresql documentation or are notes for context, so I'm hesitant to remove them.  But of course I will do it if you are sure that's what you want.

Reply:
yep - that should be updated with the corresponding notes for 13.
https://www.postgresql.org/docs/13/libpq-ssl.html